### PR TITLE
Update message for people loading site list

### PR DESCRIPTION
**Pull request by wslack**
Add note that the site list flashes a "no sites" message

### CODE REVIEW

File: frontend/components/siteList/siteList.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const getSites = (sites) => {
     return (
       <div className="usa-grid">
         <h1>No sites yet.</h1>
-        <p>Add one now.</p>
+        <p>Add one now. (This message may also appear briefly before your site list loads).</p>
```
Comment (jmhooper) [inline frontend/components/siteList/siteList.js]: This works for now. Eventually we should replace with a loading state (#832).

Comment (wslack) [inline frontend/components/siteList/siteList.js]: yep; it was loading slowly for me when tethered so I did a quick fix for the moment

**Comment by brandocalrissian** [comment]
Should we merge this puppy?

**Comment by jmhooper** [comment]
Yep!

**Pull request merged**